src/wp-includes/option.php

```diff
@@ -643,50 +643,150 @@ function wp_load_alloptions( $force_cache = false ) {
 }
 
 /**
- * Loads and primes caches of certain often requested network options if is_multisite().
+ * Primes specific network options for the current network into the cache with a single database query.
  *
- * @since 3.0.0
- * @since 6.3.0 Also prime caches for network options when persistent object cache is enabled.
+ * Only network options that do not already exist in cache will be loaded.
+ *
+ * If site is not multisite, then call wp_prime_option_caches().
+ *
+ * @since 6.6.0
+ *
+ * @see wp_prime_network_option_caches()
+ *
+ * @param string[] $options An array of option names to be loaded.
+ */
+function wp_prime_site_option_caches( array $options = array() ) {
+	wp_prime_network_option_caches( null, $options );
+}
+
+/**
+ * Primes specific network options into the cache with a single database query.
+ *
+ * Only network options that do not already exist in cache will be loaded.
+ *
+ * If site is not multisite, then call wp_prime_option_caches().
+ *
+ * @since 6.6.0
  *
  * @global wpdb $wpdb WordPress database abstraction object.
  *
- * @param int $network_id Optional. Network ID of network for which to prime network options cache. Defaults to current network.
+ * @param int      $network_id ID of the network. Can be null to default to the current network ID.
```

Comment:
Related to my previous suggestion. If we're enforcing `int` values, Let's update the docs as well.
```suggestion
 * @param int      $network_id ID of the network. Passing 0 will default to the current network ID.
```